src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -7,26 +7,29 @@
 import mate.academy.hibernate.relations.service.ActorService;
 import mate.academy.hibernate.relations.service.CountryService;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.service.impl.ActorServiceImpl;
+import mate.academy.hibernate.relations.service.impl.CountryServiceImpl;
+import mate.academy.hibernate.relations.service.impl.MovieServiceImpl;
 import mate.academy.hibernate.relations.util.HibernateUtil;
 import org.hibernate.SessionFactory;
 
 public class Main {
     public static void main(String[] args) {
         // use this session factory when you will initialize service instances
-        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        SessionFactory sessionFactory = HibernateUtil.getInstance();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(sessionFactory);
```

Comment:
```suggestion
        CountryService countryService = new CountryServiceImpl(new CountryDaoImpl(sessionFactory));
```

same for other services

Reply:
Why need to do this? Why my option is not correct?

Reply:
You should provide isolation and abstraction and dep injection
Why do you need to make such logic in service?


```
public ActorServiceImpl(SessionFactory sessionFactory) {
        actorDao = new ActorDaoImpl(HibernateUtil.getInstance());
    }
```